src/ui.button/configs/default.config.js

```diff
@@ -51,62 +50,38 @@ export default /*tw*/ {
       },
     },
     compoundVariants: [
-      {
-        disabled: true,
-        variant: "primary",
-        class: `
-          border-opacity-0 bg-opacity-40 cursor-no-drop text-white
-          hover:bg-opacity-40
-          focus:bg-opacity-40 focus:ring-0
-          active:bg-opacity-40 active:ring-0
-        `,
-      },
-      {
-        disabled: true,
-        variant: "secondary",
-        class: `
-          text-opacity-40 border-opacity-40 cursor-no-drop
-          hover:text-opacity-40 hover:border-opacity-40
-          focus:text-opacity-40 focus:border-opacity-40 focus:ring-0
-          active:text-opacity-40 active:border-opacity-40 active:ring-0
-        `,
-      },
-      {
-        disabled: true,
-        variant: "thirdary",
-        class: `
-          bg-opacity-5 text-opacity-40 ring-opacity-0 cursor-no-drop
-          hover:bg-opacity-5 hover:text-opacity-40
-          focus:ring-0 focus:bg-opacity-5 focus:text-opacity-40
-          active:bg-opacity-5 active:ring-0 active:text-opacity-40
-        `,
-      },
       {
         color: "grayscale",
         variant: "primary",
         class: `
           border-gray-900 bg-gray-900
-          hover:border-gray-900 hover:bg-gray-900
-          focus:border-gray-900  focus:ring-gray-900/20
-          active:border-gray-900 active:bg-gray-900`,
+          hover:border-gray-700 hover:bg-gray-700
+          focus:border-gray-700  focus:ring-gray-700/20
+          active:border-gray-600 active:bg-gray-600
+          disabled:bg-gray-500 disabled:ring-0 disabled:cursor-no-drop
```

Comment:
`disabled:ring-0 disabled:cursor-no-drop` may be moved to the base classes to prevent duplications